extensions/panache/panache-common/deployment/src/main/java/io/quarkus/panache/common/deployment/JandexUtil.java

```diff
@@ -346,6 +347,23 @@ private static void unbox(MethodVisitor mv, String owner, String methodName, Str
         mv.visitMethodInsn(Opcodes.INVOKEVIRTUAL, owner, methodName, "()" + returnTypeSignature, false);
     }
 
+    public static MethodInfo findMethod(ClassInfo classInfo, MethodInfo methodInfo) {
```

Comment:
This can't work, as it ignores type arguments. I've already removed this method two times because it's incorrect and lures people into thinking it will work when it does not. In both cases I didn't need that method.

Reply:
No longer used.  removed.